src/main/resources/camunda/camunda-task-clean-up-query.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
could we add an integration test to verify this SQL query will behave as expected?